### DISCUSSION TITLE
[@types/jest-when] - Remove jest type reference

### DISCUSSION THREAD

**Discussion by MWhite-22** [DefinitelyTyped/DefinitelyTyped#64465] [Issues with a @types package]
Is it possible to remove the jest types reference from this @type pkg? 

The reference to jest types is overriding the Vitest global types. Vitest is fully functioning with the jest-when library, just not with the @types/jest-when definitions.

[a comment by an automated account is not shown]

**Comment by MWhite-22** [comment]
@aldentaylor, @immanuel192, @gstamac, @HipsterBrown, @boginw

Any objection to me submitting a PR to remove the reference?

**Reply by boginw** [reply]
If it can be done without complicating the types too much, and if it can be done while retaining backward compatibility, then I cannot see a reason why not. Just keep in mind, the library is called *jest*-when, so it is not unnatural to depend on `jest` here.

Do you know of any other libraries which have similar issues with `jest` vs. `vitest` typings? How have they tackled issues like these?

**Reply by MWhite-22** [reply]
Here is an example of vitest implementation of another jest specific package. https://github.com/testing-library/jest-dom/issues/439

And the DefinitelyTyped PR that made it possible is here: #59030

Ill throw a sample PR up in a few and test it all out.
